buildenvs/github-action.Dockerfile

```diff
@@ -106,10 +42,12 @@ RUN set -xe; \
       ca-certificates \
       flex \
       gawk \
+      gcc-12 \
+      gcc-12-aarch64-linux-gnu \
+      gcc-12-arm-linux-gnueabihf \
       git \
       libncursesw5 \
       libncursesw5-dev \
```

Comment:
If we refactor we should set the versions also

Reply:
What do you mean?